modules/installation-gcp-permissions.adoc

```diff
@@ -40,6 +301,60 @@ ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
 * Service Account Key Admin
+
+The following are the additional permissions required for user-provisioned GCP infrastructure for creating and deleting the {product-title} cluster.
+
+.Required Images permissions
+[%collapsible]
+====
+* `compute.images.create`
+* `compute.images.delete`
+* `compute.images.get`
+* `compute.images.list`
+====
+
+.Required permissions to get Region related information
+[%collapsible]
+====
+* `compute.regions.get`
+====
+
+.Required Deployment Manager permissions
+[%collapsible]
+====
+* `deploymentmanager.deployments.create`
+* `deploymentmanager.deployments.delete`
+* `deploymentmanager.deployments.get`
+* `deploymentmanager.deployments.list`
+* `deploymentmanager.manifests.get`
+* `deploymentmanager.operations.get`
+* `deploymentmanager.resources.list`
+====
+
+.Required Service Account permissions
+[%collapsible]
+====
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccountKeys.get`
+* `iam.serviceAccountKeys.list`
+* `iam.serviceAccounts.actAs`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.serviceAccounts.list`
+* `resourcemanager.projects.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+.Required permissions for deleting Deployment Manager
+[%collapsible]
+====
+* `deploymentmanager.deployments.delete`
+* `deploymentmanager.deployments.list`
+* `deploymentmanager.operations.get`
```

Comment:
```suggestion
* `deploymentmanager.operations.get`
* `deploymentmanager.deployments.list`
```
Rearranging the permissions in ascending order

Reply:
Addressed